Project.toml

```diff
@@ -6,7 +6,9 @@ version = "0.17.3"
 [deps]
 ArrayLayouts = "4c555306-a7a7-4459-81d9-ec55ddd5c99a"
 BandedMatrices = "aae01518-5342-5314-be14-df237901396f"
+CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
 FastGaussQuadrature = "442a2c76-b920-505d-bb47-c5924d526838"
+ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
```

Comment:
Also add an entry to the `[compat]` section further below. For example `ForwardDiff = "0.10"`.

Reply:
Done.